### ISSUE TITLE
We should create a demo page

### ISSUE THREAD

**Issue by rmhrisk** [PeculiarVentures/xadesjs#17]
I have xadesjs.com pointed at our git repository here, I would like for us to build a demo application and host it on git.

Here is a proposal for what that might look like:
https://docs.google.com/document/d/1MIK7t7VHOz7OxzO7k0nOzlabjikdI4RkAkpedNU48JU/edit#heading=h.tb2kpesf0b6j

**Comment by bjrmatos** [comment]
and in node.js providing demos with https://tonicdev.com/ would be a killer way to play with the API!

**Comment by rmhrisk** [comment]
Thats cool, I like the idea of being able to fiddle right in the page.

**Comment by rmhrisk** [comment]
One thought since the code in browser and in node is the same, maybe we wouldn't need tonic? Just a run button that does an eval of the code?

**Comment by bjrmatos** [comment]
@rmhrisk yes, but a nice feature of tonic is the it lets to play with different versions of node :)

**Comment by rmhrisk** [comment]
we now have a very basic demo page at xadesjs.com, not as complete as suggested in this thread but enough to show basic functionality.
